spec/models/visit_spec.rb

```diff
@@ -3,31 +3,31 @@
 RSpec.describe Visit, type: :model do
   describe 'does not pass validations with' do
     context 'nil start_date' do
-      subject { FactoryBot.build(:random_visit, :nil_start_at) }
+      subject { build(:random_visit, :nil_start_at) }
```

Comment:
I prefer explicit FactorBot. Otherwise set it as a defined code style within whole team.